docs/src/fs.rst

```diff
@@ -232,6 +249,8 @@ API
     Equivalent to :man:`preadv(2)`. If the `offset` argument is `-1`, then
     the current file offset is used and updated.
 
+    The result in `req->result` indicates the amount of bytes read, if > 0.
```

Comment:
I realize this is nitpicking but isn't it correcter to say:
```suggestion
    The result in `req->result` indicates the number of bytes read, if > 0.
```
Same on line 268. Maybe also mention that 0 means EOF.